qemu/qemu.go

```diff
@@ -2568,6 +2619,19 @@ func (config *Config) appendLogFile() {
 	}
 }
 
+func (config *Config) appendFwCfg(logger QMPLog) {
+	if logger == nil {
+		logger = qmpNullLogger{}
+	}
+
+	if !config.FwCfg.Valid() {
+		logger.Errorf("fw_cfg is not valid")
```

Comment:
I'd change this to:


```go
logger.WithField("fwcfg", config.FwCfg).Error("invalid fw_cfg")
```

Reply:
Sure. going to change that

Reply:
hmm but logger does not have `WithField` available. that comes from `logrus`

Reply:
Ah - sorry. I'm so used to using the *de facto* now ;). In which case, just log the `FwCfg` so the user has an indication as to what is not valid about it (or change the `Valid()` func to return an `error` rather than a `bool`.